specs/commandline/certificate.md

```diff
@@ -0,0 +1,247 @@
+
+Flags:
+  -h, --help   help for certificate
+
+Global Flags:
+      --plain-http   Registry access via plain HTTP
+```
+
+### notation certificate add
+
+```text
+Add certificates to the trust store. This command only operates on User level.
+
+Usage:
+  notation certificate add --type <type> --store <name> <filepath...> [flags]
+
+Flags:
+  -h, --help   Help for certificate
+  -s, --store string   Specify named store
+  -t, --type string    Specify trust store type, options: ca, signingAuthority
+
+Global Flags:
+      --plain-http   Registry access via plain HTTP
+```
+
+### notation certificate list
+
+```text
+List certificates in the trust store. This command operates on both User level and System level.
+
+Usage:
+  notation certificate list [flags]
+
+Aliases:
+  list, ls
+
+Flags:
+  -h, --help           help for list
+  -s, --store string   specify named store
+  -t, --type string    specify trust store type, options: ca, signingAuthority
+
+Global Flags:
+      --plain-http   Registry access via plain HTTP
+```
+
+### notation certificate show
+
+```text
+Show certificate details given trust store type, named store, and certificate file name. If the certificate file contains a certificate chain, all certificates in the chain are displayed starting from the leaf. User level has priority over System level.
+
+Usage:
+  notation certificate show -t <type> -s <name> <fileName> [flags]
+
+Flags:
+  -h, --help           help for show
+  -s, --store string   specify named store
+  -t, --type string    specify trust store type, options: ca, signingAuthority
+
+Global Flags:
+      --plain-http   Registry access via plain HTTP
+```
+
+### notation certificate delete
+
+```text
+Delete certificates from the trust store. This command only operates on User level.
+
+Usage:
+  notation certificate delete -t <type> -s <name> (--all | <cert_filename>) [flags]
+
+Aliases:
+  delete, rm
+
+Flags:
+  -a, --all            If set to true, remove all certificates in the named store
+  -y, --confirm        If yes, do not prompt for confirmation of deletion
+  -h, --help           help for delete
+  -s, --store string   Specify named store
+  -t, --type string    Specify trust store type, options: ca, signingAuthority
+
+Global Flags:
+      --plain-http   Registry access via plain HTTP
+```
+
+### notation certificate generate-test
+
+```text
+Generate a test RSA key and a corresponding self-signed certificate.
+
+Usage:
+  notation certificate generate-test <host> [flags]
+
+Flags:
+  -b, --bits int      RSA key bits (default 2048)
+  -d, --default       mark as default
+  -h, --help          help for generate-test
+  -n, --name string   key and certificate name
+      --trust         add the generated certificate to the trust store
```

Comment:
Since we have trust policy and trust store, we need to elaborate what `--trust` means.

Reply:
I think the description is clear for `--trust`. Do you mean we need to change the name of `--trust` for trust store only? For example `--trust-store`?

Reply:
This flag only adds the generated certificate to the trust stone, no operations on trust policy.